systemd/docker-novolume-plugin.socket

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Docker novolume plugin Socket for the API
+#PartOf=docker-novolume-plugin.service
+
+[Socket]
+ListenStream=/run/docker/plugins/docker-novolume-plugin.sock
+SocketMode=0660
+SocketUser=root
+SocketGroup=docker
```

Comment:
Yes we do not want this to be accessible by docker group.  It should probably be 0700 and only accessible from root.